### PR TITLE
Added new version folder 1 with new docker and rancher compose file

### PR DESCRIPTION
**Pull request by sensri108**
Add new version folder 1 with new docker and rancher compose the file.

### CODE REVIEW

File: infra-templates/portworx/1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+portworx:
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.hostname_override: container_name
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
+    io.rancher.container.pull_image: always
+  image: portworx/px-dev:edge
```
Comment (rawmind0) [inline infra-templates/portworx/1/docker-compose.yml]: Please, use concrete image version instead generic label. It's not reproducible.
Also, pull image always is not needed.
```
-    io.rancher.container.pull_image: always
-  image: portworx/px-dev:edge
+  image: portworx/px-dev:1.2.11.1
```

File: infra-templates/portworx/1/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      required: true
+      default: "/usr/src"
+
+
+portworx:
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.hostname_override: container_name
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
+    io.rancher.container.pull_image: always
```
Comment (rawmind0) [inline infra-templates/portworx/1/rancher-compose.yml]: Don't needed labels here, also added at docker-compose.
```
-  labels:
-    io.rancher.container.dns: 'true'
-    io.rancher.container.hostname_override: container_name
-    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
-    io.rancher.container.pull_image: always
```

File: infra-templates/portworx/1/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.catalog:
+  name: "Portworx"
+  version: "1.2.0"
```
Comment (rawmind0) [inline infra-templates/portworx/1/rancher-compose.yml]: version field should match version field at infra-templates/portworx/config.yml
```
-  version: "1.2.0"
+  version: "1.2.11.1"
```

**Review by rawmind0 — CHANGES_REQUESTED** [review]

File: infra-templates/portworx/1/rancher-compose.yml (hunk 3 above)
Comment (sensri108) [reply infra-templates/portworx/1/rancher-compose.yml]: I will update the same. thanks

File: infra-templates/portworx/1/rancher-compose.yml (hunk 2 above)
Comment (sensri108) [reply infra-templates/portworx/1/rancher-compose.yml]: got it. i will validate it, untill then let it be as it is for now.

File: infra-templates/portworx/1/docker-compose.yml (hunk 1 above)
Comment (sensri108) [reply infra-templates/portworx/1/docker-compose.yml]: FYI. Currently, both build images are same( edge and 1.2.11.1). I will update appropriately.  pull image always is needed, for now. we continuously updating images and tagging to edge. thanks

Comment (rawmind0) [reply infra-templates/portworx/1/docker-compose.yml]: That's the point, today edge and 1.2.11.1 are the same but may be tomorrow not, edge is not repoducible.  Please, use concrete image tag and you don't needed pull image always.

Comment (sensri108) [reply infra-templates/portworx/1/docker-compose.yml]: got it, it's going to be always 'edge'.  Do you want me to remove rancher label? (-    io.rancher.container.pull_image: always) and resubmit pull request again?

**Comment by sensri108** [comment]
I am still wondering, what else needs to be updated to get it approved. I would like to discuss at the meeting. Can you please send me to invite based on your availability tomorrow?. I need this template to be updated ASAP.

File: infra-templates/portworx/1/docker-compose.yml (hunk 1 above)
Comment (rawmind0) [reply infra-templates/portworx/1/docker-compose.yml]: Again, `edge` neither generic labels are not reproducible. Catalog package manage version, if you are deploying `1.2.11.3`, please use concrete docker label.

File: infra-templates/portworx/1/rancher-compose.yml (hunk 2 above)
Comment (rawmind0) [reply infra-templates/portworx/1/rancher-compose.yml]: Please, review and validate it before approve.

File: infra-templates/portworx/1/docker-compose.yml (hunk 1 above)
Comment (sensri108) [reply infra-templates/portworx/1/docker-compose.yml]: ok, Updated as you mentioned. Please review and approve it asap. thanks

File: infra-templates/portworx/1/rancher-compose.yml (hunk 2 above)
Comment (sensri108) [reply infra-templates/portworx/1/rancher-compose.yml]: Sure, Who needs to review and validate? is it someone within portworx? or in rancher?

Comment (rawmind0) [reply infra-templates/portworx/1/rancher-compose.yml]: Please remove labels from rancher-compose, already added to docker-compose

Comment (rawmind0) [reply infra-templates/portworx/1/rancher-compose.yml]: Please remove labels from rancher-compose, already added to docker-compose

**Review by sensri108 — COMMENTED** [review]
removed duplicate lables

File: infra-templates/portworx/1/rancher-compose.yml (hunk 2 above)
Comment (sensri108) [reply infra-templates/portworx/1/rancher-compose.yml]: removed labels from rancher-compose.

**Review by sensri108 — COMMENTED** [review]
Updated all changes and reviewed.

**Review by rawmind0 — APPROVED** [review]

**Comment by rawmind0** [comment]
LGTM..

**Review by sensri108 — COMMENTED** [review]
all changes completed.

**Comment by sensri108** [comment]
Thank you .

**Comment by sensri108** [comment]
Hope all this changes will be merged to public portworx catalog? 

 

Sen S

**Comment by will-chan** [comment]
LGTM

**Review by will-chan — APPROVED** [review]
LGTM

**Pull request merged**